demo/hyproof-demo.md

```diff
@@ -0,0 +1,183 @@
+
+We will now walk through the demonstration following a *happy path* workflow. This takes the following steps:
+
+- Heidi the Hydrogen Producer initialises a certificate
+- Emma the Energy Supplier calculates the embodied CO~2~ (eCO~2~)
+- Emma the Energy Supplier adds eCO~2~ to the certificate
+- The system now issues the final certificate
+
+One further workflow can be demonstrated for Reginald the Regulator:
+
+- Reginald the Regulator reviews and revokes a certificate
+
+### 1. Heidi the Hydrogen Producer initialises a certificate
+
+You will first adopt the persona of Heidi, a Hydrogen Producer. Heidi has just generated a batch of hydrogen through electrolysis and is now seeking to have that batch certified.
+
+
+
+In order to obtain a certificate for a new batch of hydrogen, click on the `New Certificate` button in the right hand panel.
+
+This will bring up an input page where Heidi records the start and end time of hydrogen production, the batch size to be certified, and the amount of electricity that was used to generate the hydrogen.
+
+>image of data entry page
+
+When you've finished entering all of the relevant information, click the `Submit` button in the right hand panel.
+
+The system will now send this data to the shared ledger via the API and display a small wait animation in the button.
+
+`Heidi` also transmits a separate packet of data to her contracted energy provider `Emma the Energy Supplier` to allow `Emma` to complete the certification process.
+
+The panel on the left hand side of the screen will indicate when the certificate has been successfully `Initiated` on the ledger.
+
+>image of left panel showing initiated
+
+It is now time to change persona to `Emma the Energy Supplier`. 
+
+### 2. Emma the Energy Supplier completes the certificate
+
+You should first change persona by pressing the right arrow just below the `Hydrogen Innovation Initiative` logo in the top left hand side of the screen.
+
+>Image showing arrow highlighted
+
+This opens a drawer showing the 4 personas available in the demonstrator. Select `Emma` and the background will turn green indicating a successful switch. Close the drawer by pressing the arrow again.
+
+`Emma`'s work is automatically carried out by the system itself in the background. 
+
+>image of emma's system in busy state
+
+What is happening is as follows:
+
+1. `Emma` previously received a data packet from `Heidi` containing:
+
+    - a `cryptographic commitment` to the content of this data
+    - the `start time` and `end time` of `Heidi's` hydrogen production
+    - the total `kWh` of electricity used by `Heidi`
+
+2. `Emma` first checks this additional private data aligns with what has been publicly recorded by `Heidi` on the ledger using the `cryptographic commitment`. This keeps `Heidi` honest
+
+3. `Emma` then calculates the embodied CO~2~ for `Heidi`'s batch from:
```

Comment:
Is `CO~2~` a special scientific thing ? for some reason this renders as ~~2~~ on my screen ..

Reply:
Trying to use the tildes ~ to render subscript (read online that's how to do it in md) but seems dependent on the particular md renderer. Might just leave it as CO2